src/components/sbb-notification/sbb-notification.tsx

```diff
@@ -0,0 +1,260 @@
+  @Prop() public titleContent?: string;
+
+  /**
+   * Level of title, will be rendered as heading tag (e.g. h3). Defaults to level 3.
+   */
+  @Prop() public titleLevel: InterfaceTitleAttributes['level'] = '3';
+
+  /**
+   * Whether the notification is readonly.
+   * In readonly mode, there is no dismiss button offered to the user.
+   */
+  @Prop({ reflect: true }) public readonly = false;
+
+  /**
+   * Whether the animation is enabled.
+   */
+  @Prop({ reflect: true, mutable: true }) public disableAnimation = false;
+
+  /**
+   * State of listed named slots, by indicating whether any element for a named slot is defined.
+   */
+  @State() private _namedSlots = createNamedSlotState('title');
+
+  /**
+   * The state of the notification.
+   */
+  @State() private _state: InterfaceNotificationAttributes['state'] = 'opened';
+
+  @State() private _iconName: NotificationIconName = NotificationIconName[this.type];
+
+  @State() private _hasTitle = false;
+
+  @State() private _currentLanguage = documentLanguage();
+
+  @State() private _resizeDisableAnimation = false;
+
+  @Element() private _element!: HTMLElement;
+
+  private _notificationElement: HTMLElement;
+  private _resizeObserverTimeout: ReturnType<typeof setTimeout> | null = null;
+  private _notificationResizeObserver = new ResizeObserver(() => this._onNotificationResize());
+
+  @Watch('type')
+  public updateNotificationState(newValue: InterfaceNotificationAttributes['type']): void {
+    this._iconName = NotificationIconName[this.type];
+    this.type = newValue ?? 'info';
+  }
+
+  /**
+   * Emits whenever the notification starts the opening transition.
+   */
+  @Event({
+    bubbles: true,
+    composed: true,
+    eventName: 'will-open',
+  })
+  public willOpen: EventEmitter<void>;
+
+  /**
+   * Emits whenever the notification is opened.
+   */
+  @Event({
+    bubbles: true,
+    composed: true,
+    eventName: 'did-open',
+  })
+  public didOpen: EventEmitter<void>;
+
+  /**
+   * Emits whenever the notification begins the closing transition.
+   */
+  @Event({
+    bubbles: true,
+    composed: true,
+    eventName: 'will-close',
+  })
+  public willClose: EventEmitter<void>;
+
+  /**
+   * Emits whenever the notification is closed.
+   */
+  @Event({
+    bubbles: true,
+    composed: true,
+    eventName: 'did-close',
+  })
+  public didClose: EventEmitter<void>;
+
+  @Method()
+  public async close(): Promise<void> {
+    if (this._state === 'opened') {
+      this._state = 'closing';
+      this.willClose.emit();
+      this.disableAnimation && this._handleClosing();
+    }
+  }
+
+  private _handlerRepository = new HandlerRepository(
+    this._element,
+    languageChangeHandlerAspect((l) => (this._currentLanguage = l)),
+    namedSlotChangeHandlerAspect((m) => (this._namedSlots = m(this._namedSlots)))
+  );
+
+  public connectedCallback(): void {
+    this._handlerRepository.connect();
+    this._hasTitle = !!this.titleContent || this._namedSlots['title'];
```

Comment:
Slotting title at a later point doesn’t seem to be supported. I suggest to check it only in render as in other components.